spec/spec_helper.rb

```diff
@@ -16,6 +16,8 @@
 SimpleCov.minimum_coverage 90
 SimpleCov.start 'rails'
 
+require 'pry'
```

Comment:
When running the spec in isolation it wasn't working, which was because the spec loaded `spec_helper`, and meant the `helper` helper method wasn't available. The spec now uses `rails_helper`. I will make that line conditional with a `DEBUG` env; it is handy in my experience when adding `binding.pry` breakpoints. Would that be acceptable?

Reply:
OK. I've moved over to `binding.irb` which means I don't need to use Pry at all in my everyday use. So I'm happy to leave this as is.

Reply:
Try it and see :smile:

Reply:
The answer is.... no

Reply:
:smile: OK